app/facades/food_facade.rb

```diff
@@ -0,0 +1,15 @@
+class FoodFacade
+
+  def self.service(food)
+    FoodService.get_details(food)
+  end
+
+  def self.food_info(info)
+    food = service(info)
+
+    food.map do |food_data|
```

Comment:
Would have liked to see you do a `food.first(10).map...` so that we only ever create 10 objects instead of thousands. This is a big performance hit.